chrome_manifest_v3/service_worker.js

```
@@ -1,79 +1,79 @@
  */
 function logbs(data) {
-    if (typeof debugMode_g === "undefined") {
-        chrome.storage.sync.get("debugMode", function (response) {
-            if (response.hasOwnProperty("debugMode") && response["debugMode"]) {
-                debugMode_g = true;
-            }
-        });
-    }
+  if (typeof debugMode_g === 'undefined') {
+    chrome.storage.sync.get('debugMode', function (response) {
+      if (response.hasOwnProperty('debugMode') && response['debugMode']) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)